.github/workflows/build.yaml

```diff
@@ -286,9 +243,9 @@ jobs:
     - name: Restore vcpkg and its artifacts.
       uses: actions/cache@v3
       with:
-        path: ${{ env.VCPKG_DEFAULT_BINARY_CACHE }}
+        path: ${{ env.VCPKG_ROOT }}/installed
         key: |
-          ${{needs.vcpkg-packages.outputs.vcpkg_key}}-${{ matrix.arch }}
+          ${{ hashFiles( './install-dependencies.sh', '.github/scripts/vcpkg_ref' ) }}-${{ matrix.arch }}
```

Comment:
Maybe instead of hashing the stored vcpkg_ref file, the commit ID from it could be included in the key directly, so the caches could be easily identified.

Reply:
I restored the previous key format